config/bidderinfo_test.go

```diff
@@ -2008,3 +2008,97 @@ func TestReadFullYamlBidderConfig(t *testing.T) {
 	}
 	assert.Equalf(t, expectedBidderInfo, actualBidderInfo, "Bidder info objects aren't matching")
 }
+
+func TestValidateGeoscope(t *testing.T) {
+	testCases := []struct {
+		name       string
+		geoscope   []string
+		bidderName string
+		expectErr  bool
+		errMsg     string
+	}{
+		{
+			name:       "nil-geoscope",
```

Comment:
Nitpick: The word "geoscope" is redundant. Consider calling this "nil" and the other "empty".